packages/user/.env

```
@@ -0,0 +1,14 @@
+# {
+#     "tokens": {
+#         "access_token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoiOTNkMDE2YjUtOTQ2Ny0xMWYwLWEzYzktNTRlZTc1NmQ4OTUyIiwidXNlcm5hbWUiOiJpYW4gbXdhbmdpIiwiZW1haWwiOiJpYW5td2ExNDNAZ21haWwuY29tIiwicm9sZXMiOlsidXNlciJdLCJpc3MiOiJjaGF0Iiwic3ViIjoidXNlci10b2tlbiIsImF1ZCI6WyJjaGF0Il0sImV4cCI6MTc1ODE5NjMxOCwibmJmIjoxNzU4MTkyNzE4LCJpYXQiOjE3NTgxOTI3MTgsImp0aSI6IjRiYjBiNzVjLWY2YjgtNDQ2Ni1iNmYxLTA5OGQzYWIwMDhiYyJ9.1vuw3N0TH1EPvx8jUflHAEFD0LxF6zjNyXKyeydtQb0",
+#         "refresh_token": "891c513a1af37b0159916da5733bc04131a9527e8cad7389488dfb858270fc24",
+#         "expires_at": {
+#             "seconds": "1758196318",
+#             "nanos": 358319600
+#         }
+#     }
+# }
+
+# 93d016b5-9467-11f0-a3c9-54ee756d8952
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)